flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/serde/RexNodeJsonSerializer.java

```diff
@@ -397,6 +398,9 @@ static void serializeSqlOperator(
             // stack is exposed to the user and can thus be external.
             gen.writeStringField(
                     FIELD_NAME_INTERNAL_NAME, BuiltInSqlOperator.toQualifiedName(operator));
+            if (operator.getName().isEmpty()) {
```

Comment:
Why does `FIELD_NAME_SQL_KIND` field need the check whether the operator name is empty?